src/test/features/dashboard/routes/claimStatus/full-defence.ts

```diff
@@ -0,0 +1,410 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from 'test/routes/hooks'
+import 'test/routes/expectations'
+
+import { Paths } from 'dashboard/paths'
+
+import { app } from 'main/app'
+
+import * as idamServiceMock from 'test/http-mocks/idam'
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+import { checkAuthorizationGuards } from 'test/features/dashboard/routes/checks/authorization-check'
+import { MomentFactory } from 'shared/momentFactory'
+import { FreeMediationOption } from 'forms/models/freeMediation'
+
+import {
+  baseDefenceData,
+  baseResponseData,
+  defenceWithAmountClaimedAlreadyPaidData
+} from 'test/data/entity/responseData'
+
+import {
+  respondedAt,
+  claimantRejectAlreadyPaid,
+  directionsQuestionnaireDeadline,
+  settlementOffer,
+  settlementOfferAccept,
+  settlementOfferReject,
+  settledWithAgreement
+} from 'test/data/entity/fullDefenceData'
+
+const cookieName: string = config.get<string>('session.cookieName')
+
+const fullDefenceClaim = {
+  ...claimStoreServiceMock.sampleClaimObj,
+  responseDeadline: MomentFactory.currentDate().add(1, 'days'),
+  response: {
+    ...baseResponseData,
+    ...baseDefenceData,
+    amount: 30
+  },
+  ...respondedAt
+}
+
+const testData = [
+  {
+    status: 'Full defence - defendant paid what he believe',
+    claim: fullDefenceClaim,
+    claimOverride: {
+      response: { ...defenceWithAmountClaimedAlreadyPaidData }
+    },
+    claimantAssertions: [
+      'The defendant’s response',
+      fullDefenceClaim.claim.defendants[0].name + ' believes that they’ve paid the claim in full.',
+      'Respond'
+    ],
+    defendantAssertions: [
+      'Your response to the claim',
+      'We’ve emailed ' + fullDefenceClaim.claim.claimants[0].name + ' telling them when and how you said you paid the claim.',
+      'Download your response'
+    ]
+  },
+  {
+
+    status: 'Full defence - defendant paid what he believe - claimant rejected defendant response',
+    claim: fullDefenceClaim,
+    claimOverride: {
+      response: { ...defenceWithAmountClaimedAlreadyPaidData },
+      ...claimantRejectAlreadyPaid,
+      ...directionsQuestionnaireDeadline
+    },
+    claimantAssertions: [
+      'You’ve rejected the defendant’s admission',
+      'They said they owe £' + defenceWithAmountClaimedAlreadyPaidData.paymentDeclaration.paidAmount,
```

Comment:
Prefer to use interpolation instead of concatenation.  Also instead of hard coding currency symbol, please use FormatMoney utility.

Reply:
Done, changed and committed